README.md

```diff
@@ -20,15 +26,47 @@ Simply run:
 $ inspec plugin install train-winrm
 ```
 
-You can then run:
+You can then run, using Chef InSpec as an example:
+
+```
+you@home $ inspec some-profile winrm://someuser@somehost --password somepassword
+```
 
+From Ruby code, you may use this plugin as follows:
 ```
-TODO - example
+require 'train'
+transport = Train.create('winrm',
+  host: '1.2.3.4', user: 'Administrator', password: '...', ssl: true, self_signed: true)
+conn = transport.connection
 ```
 
 ## Target Options for Train-WinRM
 
-TODO
+### host
+
+Required `String`. The hostname or IP address to connect to.
+
+#### port
+
+Optional `Integer`, default 5985 (plain) or 5896 (SSL). The port number to which the connection should be made.
+
+### user
+
+Optional `String`, username to connect as.  Default 'Administrator'.
```

Comment:
```suggestion
Optional `String`. Username used for sign in.  Default: 'Administrator'.
```

Reply:
I'm ambivalent about using "sign in" vs. "connection" at the moment. I am more concerned about including a verb in the description sentence for your target options.